deployments/helm/nvidia-device-plugin/templates/daemonset-mps-control-daemon.yml

```diff
@@ -0,0 +1,224 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+{{- $hasConfigMap := (include "nvidia-device-plugin.hasConfigMap" .) | trim }}
```

Comment:
Can you also update the example MPS config [here](https://github.com/NVIDIA/k8s-device-plugin/blob/main/deployments/helm/nvidia-device-plugin/values.yaml#L1).

Reply:
Sure, I can add an MPS config. As a matter of interest, since there are very few other config options show there is there any guidance on when we show examples there and when we don't?